src/SearchViewProvider.ts

```diff
@@ -152,27 +129,21 @@ export default class SearchViewProvider implements vscode.WebviewViewProvider {
                     <div class="loader js-loader">
                         <div class="loaderBar"></div>
                     </div>
-                    <div class="filters">
-                        <label>
-                            <input type="radio" name="filterBy" value="upcoming" checked class="js-filter" />
-                            current
-                        </label>
-                        <label>
-                            <input type="radio" name="filterBy" value="current" class="js-filter" />
-                            upcoming
-                        </label>
-                    </div>
                     <input 
                         class="js-query-input" 
-                        type="text" 
-                        title="Wildcards: query*   s*Model \nFields: title:OrderMgr \nFuzzy:  odrer~1  oderr~2 \nTerm presence: +getContent -pipelet -upcoming"
-                        placeholder="Search *query -pipelet +static fuzzy~3" />
```

Comment:
This one is probably the only "deal breaker" you might want to consider before accepting this PR.

The SFCCDocs.com API does not currently support fuzzy search logic, meaning the kinds of searches used in the placeholder and title actually won't work if you tried them.

However, the search within this API is substantially better than the "official docs" site, so theres should be WAY less noise or incorrect matches, so hopefully this is not a deal breaker ( but I understand if losing this functionality is too much for some folks ).

Just wanted to call it out either way as ... a "thing", ha ha